src/screens/NewAnalytics/Graphs/BarGraph/BarGraph.res

```diff
@@ -0,0 +1,11 @@
+external barGraphOptionsToJson: BarGraphTypes.barGraphOptions => JSON.t = "%identity"
```

Comment:
Move to identity file

Reply:
this is not used anywhere else other than here, so we can keep it here itself

Reply:
The identity file is made for that purpose to keep the identity at some place it's not about reusing

Reply:
And if we are using Identy function only then why can't we use this only

```js
Identity.genericTypeToJson
```

As this also converting to json if we are by passing types it shouldn't even matter what is it before so we can use this only

Reply:
we don't want to expose the BarGraphTypes to Identity file and this is specific to Bar graph so we decided to kept it over here itself.Later we will see whether to move or not

Reply:
Hey @JeevaRamu0104 if we are by passing types it's okay to use this. Why we are adding this then. It won't add any safety
```
Identity.genericTypeToJson
```